examples/complete/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = "~> 1.6"
```

Comment:
Not sure. It's kind of an example of a root module, right? It might be copied as is to the user's root module, so it's better to follow typical root module patterns. Do you have a strong opinion on that?

Reply:
It is a root module, but it's an interesting one where we want others to be able to run it without much work. As in they should pull it down, provide creds, and apply without having to jump through a bunch of hoops. We also want to be able to run these in CI via terratest or similar one day as well and be able to test with a matrix of TF versions. 

[Cloud Posse seems to use `>=` in their examples](https://github.com/cloudposse/terraform-aws-ecs-cluster/blob/main/examples/complete/versions.tf#L1-L10), which has me of the mind that we change it. I say we do it.

Reply:
> be able to test with a matrix of TF versions.

Yeah, this totally makes sense. Updated.